matcher/src/chars/normalize.rs

```diff
@@ -1,526 +1,972 @@
-    ('\u{0275}', 'o'), // , LATIN SMALL LETTER BARRED
-    ('\u{0279}', 'r'), // , LATIN SMALL LETTER TURNED
-    ('\u{027A}', 'r'), //  WITH LONG LEG, LATIN SMALL LETTER TURNED
-    ('\u{027B}', 'r'), //  WITH HOOK, LATIN SMALL LETTER TURNED
-    ('\u{027C}', 'r'), //  WITH LONG LEG, LATIN SMALL LETTER
-    ('\u{027D}', 'r'), //  WITH TAIL, LATIN SMALL LETTER
-    ('\u{027E}', 'r'), //  WITH FISHHOOK, LATIN SMALL LETTER
-    ('\u{027F}', 'r'), //  WITH FISHHOOK, LATIN SMALL LETTER REVERSED
-    ('\u{0280}', 'R'), // , LATIN LETTER SMALL CAPITAL
-    ('\u{0281}', 'R'), // , LATIN LETTER SMALL CAPITAL INVERTED
-    ('\u{0282}', 's'), //  WITH HOOK, LATIN SMALL LETTER
-    ('\u{0287}', 't'), // , LATIN SMALL LETTER TURNED
-    ('\u{0288}', 't'), //  WITH RETROFLEX HOOK, LATIN SMALL LETTER
-    ('\u{0289}', 'u'), //  BAR, LATIN SMALL LETTER
-    ('\u{028B}', 'v'), //  WITH HOOK, LATIN SMALL LETTER
-    ('\u{028C}', 'v'), // , LATIN SMALL LETTER TURNED
-    ('\u{028D}', 'w'), // , LATIN SMALL LETTER TURNED
-    ('\u{028E}', 'y'), // , LATIN SMALL LETTER TURNED
-    ('\u{028F}', 'Y'), // , LATIN LETTER SMALL CAPITAL
-    ('\u{0290}', 'z'), //  WITH RETROFLEX HOOK, LATIN SMALL LETTER
-    ('\u{0291}', 'z'), //  WITH CURL, LATIN SMALL LETTER
-    ('\u{0297}', 'c'), // , LATIN LETTER STRETCHED
-    ('\u{0299}', 'B'), // , LATIN LETTER SMALL CAPITAL
-    ('\u{029A}', 'e'), // , LATIN SMALL LETTER CLOSED OPEN
-    ('\u{029B}', 'G'), //  WITH HOOK, LATIN LETTER SMALL CAPITAL
-    ('\u{029C}', 'H'), // , LATIN LETTER SMALL CAPITAL
-    ('\u{029D}', 'j'), //  WITH CROSSED-TAIL, LATIN SMALL LETTER
-    ('\u{029E}', 'k'), // , LATIN SMALL LETTER TURNED
-    ('\u{029F}', 'L'), // , LATIN LETTER SMALL CAPITAL
-    ('\u{02A0}', 'q'), //  WITH HOOK, LATIN SMALL LETTER
-    ('\u{02AE}', 'h'), //  WITH FISHHOOK, LATIN SMALL LETTER TURNED
-    ('\u{0363}', 'a'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0364}', 'e'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0365}', 'i'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0366}', 'o'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0367}', 'u'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0368}', 'c'), // , COMBINING LATIN SMALL LETTER
-    ('\u{0369}', 'd'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036A}', 'h'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036B}', 'm'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036C}', 'r'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036D}', 't'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036E}', 'v'), // , COMBINING LATIN SMALL LETTER
-    ('\u{036F}', 'x'), // , COMBINING LATIN SMALL LETTER
+/// A char array corresponding to the following contiguous Unicode blocks:
+///
+/// - [Latin-1 Supplement](https://en.wikipedia.org/wiki/Latin-1_Supplement)
+/// - [Latin Extended-A](https://en.wikipedia.org/wiki/Latin_Extended-A)
+/// - [Latin Extended-B](https://en.wikipedia.org/wiki/Latin_Extended-B)
+///
+/// This covers the range `'\u{a0}'..='\u{29f}'`.
+const LATIN_1AB: [char; 512] = [
```

Comment:
```suggestion
static LATIN_1AB: [char; 512] = [
```

these should be static to ensure the compiler generates reasonable code (same for the other tables)